Kernel/Devices/IDEDiskDevice.h

```diff
@@ -53,6 +55,8 @@ class IDEDiskDevice final : public IRQHandler
     volatile bool m_interrupted { false };
     volatile u8 m_device_error { 0 };
 
+    bool m_second_drive; /** Are we referring to the second device on the channel?? */
```

Comment:
Rather than a boolean, consider if an enum wouldn't be more descriptive (exact naming can be bikeshedded, I'm not really an expert on the terminology here, just picking it out from the existing naming you used/doc).

    enum class ChannelDriveType {
        Primary,
        Secondary
    };

    ChannelDriveType m_channel_drive_type { ChannelDriveType::Primary };


Note how the documentation becomes mostly irrelevant because the type naming provides it. It's also a lot more obvious at creation time:

    create(IDEDiskDevice::ChannelDriveType::Primary);

rather than:

    create(false); /* don't create? uhh */

Reply:
> I'm not really an expert on the terminology here, just picking it out from the existing naming you used/doc).

I completely agree with this. At the time, a `bool` made sense because it's obviously only one or the other, however, it makes sense for it to be more verbose. Someone just seeing `false` would be an absolute mindfuck haha.